### PR TITLE
[4.4.3] Added image and actions to musesound update spec

### PR DESCRIPTION
**Pull request by igorkorsukov**
New spec 
```
    /*
    {
        "version": String,
        "image_url": String,  // it can be base64 data, like "data:image/png;base64,iVBORw0KGgoA......"
        "content": {
            "locale_code": {
                "notes": String,
                "features": [String]
                "action_title": String // title of action button
            }
        },

        // open app or web page url, try in order,
        // like this ["musehub://?from=musescore", "muse-hub://?from=musescore", "https://www.musehub.com"]
        "actions": {
            "windows": [String],
            "macos": [String],
            "linux": [String]
        }
    }
    */
```

### CODE REVIEW

File: src/framework/update/imusesoundscheckupdateservice.h [hunk 1]
```diff
@@ -45,7 +44,7 @@ class IMuseSoundsCheckUpdateService : MODULE_EXPORT_INTERFACE
 
     virtual Progress updateProgress() = 0;
 
-    virtual void openMuseHub() = 0;
+    virtual void openMuseHub(const std::vector<std::string>& actions) = 0;
```
Comment (RomanPudashkin) [inline src/framework/update/imusesoundscheckupdateservice.h:47]: I'm not sure we need it in this interface... Perhaps it was originally intended to be reusable, but with the addition of the actions argument, it's unclear how to use it. We can make it a private method of MuseSoundsCheckUpdateScenario

File: src/framework/update/qml/Muse/Update/MuseSoundsReleaseInfoDialog.qml [hunk 2]
```diff
@@ -78,7 +80,7 @@ StyledDialogView {
             Layout.fillWidth: true
             Layout.preferredHeight: 186
 
-            source: "qrc:/qml/Muse/Update/resources/muse_sounds_promo.png"
+            //source: "qrc:/qml/Muse/Update/resources/muse_sounds_promo.png"
```
Comment (RomanPudashkin) [inline src/framework/update/qml/Muse/Update/MuseSoundsReleaseInfoDialog.qml:83]: Maybe just remove it?

**Review by RomanPudashkin — APPROVED** [review]

File: src/framework/update/imusesoundscheckupdateservice.h (hunk 1 above)
Comment (igorkorsukov) [reply src/framework/update/imusesoundscheckupdateservice.h]: fixed

File: src/framework/update/qml/Muse/Update/MuseSoundsReleaseInfoDialog.qml (hunk 2 above)
Comment (igorkorsukov) [reply src/framework/update/qml/Muse/Update/MuseSoundsReleaseInfoDialog.qml]: fixed

**Pull request merged**